src/gui/CMakeLists.txt

```diff
@@ -238,14 +239,14 @@ if(NOT APPLE)
         endforeach(_file)
     endif()
 
-    install(FILES ${client_I18N} DESTINATION ${DATA_INSTALL_DIR}/${APPLICATION_NAME}/i18n)
+    install(FILES ${client_compiled_translations} DESTINATION ${DATA_INSTALL_DIR}/${APPLICATION_NAME}/i18n)
```

Comment:
We don't have anything to install as the resources are compiled into the binary?

Reply:
Right, thanks.